.github/workflows/tests.yml

```diff
@@ -27,13 +27,74 @@ jobs:
       - name: Rebase to current master
         run: git rebase origin/master
 
+      - name: Prepare
+        run: |
+          # # override GitHub's bind mount from host, we don't want anything from there and it interferes with ssh
+          # export HOME=$(getent passwd $(id -u) | cut -f6 -d:)
+
+          # Put the /secrets in place which we'll mount to the tasks-container
+          mkdir -p ~/secrets
+          echo '${{ secrets.COCKPITUOUS_TOKEN }}' > ~/.config/github-token
+          echo '${{ secrets.SSH_KNOWN_HOSTS }}' > ~/secrets/known_hosts
+          echo '${{ secrets.SINK_SSH_CONFIG }}' > ~/secrets/ssh-config
+          echo '${{ secrets.SINK_ID_RSA }}' > ~/secrets/id_rsa
+          echo '${{ secrets.SINK_ID_RSA_PUB }}' > ~/secrets/id_rsa.pub
```

Comment:
Let's hope we don't submit a PR which will make a mess out of our sinks :grin: However, ok for now I think -- I want to include the sink into the local deployment anyway. For the test run we then wouldn't get the `make check` on our public production logs, but that's kind of the point, and they are not interesting anyway. We can instead validate the generated test log from the local sink in the workflow.

But that's just thinking aloud. Good enough for now.

Reply:
Let's drop this again in a future iteration, I already have a "local sink" branch.